catalog/spatial/registry/pom.xml

```diff
@@ -92,6 +92,11 @@
                 <version>${project.version}</version>
             </dependency>
             <dependency>
+                <groupId>org.codice.ddf.registry</groupId>
```

Comment:
Does this project actually depend on the `registry-report-viewer`? I can't see how it would depend on one of its own submodules.